app/validators/future_date_validator.rb

```diff
@@ -3,7 +3,7 @@
 class FutureDateValidator < ActiveModel::EachValidator
   def validate_each(record, attribute, value)
     if value && value > Time.zone.now
-      record.instance_of?(::ParticipantOutcome::NPQ) ? record.errors.add(attribute, I18n.t(:future_date_outcomes, attribute:)) : record.errors.add(attribute, I18n.t(:future_date, attribute:))
```

Comment:
I think we can drop `future_date_outcomes` from the translations file

Reply:
There are also some other translations we can probably bin:

https://github.com/DFE-Digital/early-careers-framework/blob/main/config/locales/en.yml#L186
https://github.com/DFE-Digital/early-careers-framework/blob/main/config/locales/en.yml#L706

Maybe we just do a sweep of these separately though 🤔

Reply:
I think those are in the sweep later so its ok :)